### PR TITLE
[TIMOB-19763] Support item-specific peek for previewContext on ListView

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/TIMOB-19763

### CODE REVIEW

File: apidoc/Titanium/UI/iOS/PreviewAction.yml [hunk 1]
```diff
@@ -40,18 +40,29 @@ events:
     summary: Fired when the device detects a click against a preview action.
     description: |
         This event provides different properties depending on the view the <Titanium.UI.iOS.PreviewContext>
-        is assigned to. For example, the <Titanium.UI.ListView> and <Titanium.UI.TableView> provide the
-        additional properties `itemIndex` and `sectionIndex` to identify the previewed row and
-        section, respectively.
+        is assigned to. Every event has the `index` property to identify the clicked action. If the <Titanium.UI.iOS.PreviewContext>
+        is assigned to a <Titanium.UI.ListView>, the event provides the additional properties `itemIndex`, `sectionIndex`
```
Comment (pec1985) [inline apidoc/Titanium/UI/iOS/PreviewAction.yml]: `is assigned to a <Titanium.UI.ListView>` _or_  `<Titanium.UI.TableView>`

Comment (pec1985) [inline apidoc/Titanium/UI/iOS/PreviewAction.yml]: Nevermind, you removed TableView?

Comment (hansemannn) [inline apidoc/Titanium/UI/iOS/PreviewAction.yml]: Yes, dropped the legacy support, since it does not have an itemId.

**Comment by pec1985** [comment]
PR reviewed. **APPROVED!**

**Pull request merged**